### PR TITLE
New Event Type

### LINKED ISSUE #62: Failure to Process New Event Type
### Home Assistant
**Version:**
**Installation method** (according to [instructions](https://www.home-assistant.io/installation/)):
[ ] Operating System
[ ] Linux VM
[X] Container
[ ] Supervised
[ ] Core
### Integration

**Installation method:**
[X] HACS
[ ] manual from release
[ ] manual from git
**Integration version:** 
**Root topic:** 
### Application settings

**topic:**
[ ] I can see MQTT messages from application at MQTT-server
## Steps to reproduce

After upgrading to SleepAsAndroid 20231001 there appears to be a new event type that's causing the custom component to fail. 

## Debug log
```
2023-10-13 21:02:31.873 ERROR (MainThread) [homeassistant.util.logging] Exception in message_received when handling msg on 'SleepAsAndroid/Pixel 8 Pro': '{"event":"Unknown"}'

Traceback (most recent call last):

  File "/config/custom_components/sleep_as_android/__init__.py", line 253, in message_received

    target_sensor.process_message(msg)

  File "/config/custom_components/sleep_as_android/sensor.py", line 156, in process_message

    if self.state != new_state:

       ^^^^^^^^^^

  File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 569, in state

    raise ValueError(

ValueError: Sensor sensor.sleepasandroid_pixel_8_pro provides state value 'alarm_rescheduled', which is not in the list of options provided
```
## Additional information

I'm not sure, but it appears the filter for events in SleepAsAndroid doesn't work.

### PR DESCRIPTION
**Pull request by Silvenga**
This pr adds support for the new event type.

This should resolved #62.

### CODE REVIEW

**Comment by Silvenga** [comment]
I wasn't sure how to handle the other translations - so I only added the friendly text for English.

**Comment by Silvenga** [comment]
And tested on my HA instance, looks good.

File: custom_components/sleep_as_android/const.py [hunk 1]
```diff
@@ -38,4 +38,5 @@
     "sound_event_baby",
     "sound_event_laugh",
     "before_alarm",
+    "alarm_rescheduled"
```
Comment (IATkachenko) [inline custom_components/sleep_as_android/const.py:41]: Please add coma at the end of line.
Just to let future changes looks like one line changes: python is allowing this _(but JSON is not, so `en.json` will always have two lines change while adding just one)_.

**Review by IATkachenko — CHANGES_REQUESTED** [review]
Thank you for contributing!
Looks good, but small style change is needed.

File: custom_components/sleep_as_android/const.py (hunk 1 above)
Comment (Silvenga) [reply custom_components/sleep_as_android/const.py:41]: Updated.

Yeah, technically a trailing comma isn't spec'ful under JSON. 😸

Comment (Silvenga) [reply custom_components/sleep_as_android/const.py]: Oops, misread that comment - fixed and rebased.

**Review by IATkachenko — APPROVED** [review]

**Pull request merged**